write README.md

hkdnet opened the pull request:
write some rules...
- [x] wip workflow
- [x] devenv
- [x] code review
- [x] emoji

hkdnet committed:
[WIP] write README.md

hkdnet committed:
write devenv and what this app is

hkdnet committed:
write wip_workflow and code review

hkdnet committed:
write emoji

hkdnet commented:
please review:star:

keicy commented on README.md:
```diff
@@ -1,3 +1,21 @@
 # 3chLine
 
 a [Sails](http://sailsjs.org) application
```
please remove　this line.

hkdnet replied:
we talk about our app name and rename `3chLine` -> `3chPiazza`
so i'll remove l.3 and modify app name :+1:

hkdnet committed:
remove sails introduction and modify our app name: 3chPiazza

hkdnet commented:
@keicy please review :hand:

keicy commented on README.md:
```diff
@@ -1,3 +1,21 @@
+follow this guideline
+[初心者向けWIPワークフロー&レビュー時の注意](http://qiita.com/sue738/items/09d9292e7654747ca208)
+
+what is the most important is _code review_  
```
is it a little better of "The most important thing is ~" ?

hkdnet replied:
it's simple.
ok i'll modify it :runner:

keicy commented on README.md:
```diff
@@ -1,3 +1,21 @@
+[初心者向けWIPワークフロー&レビュー時の注意](http://qiita.com/sue738/items/09d9292e7654747ca208)
+
+what is the most important is _code review_  
+we review our code and make our app cool :smiley:
```
"We review our code each other and ~" is a litter better?

hkdnet committed:
Update README.md

hkdnet commented:
Modify a phrase and the first characters each line are replaced with upper case.

keicy commented on README.md:
```diff
@@ -1,3 +1,19 @@
+# 3chPiazza
 
-a [Sails](http://sailsjs.org) application
+* tw*tter clone for 3ch
```
"Communication "Piazza" for 3ch."  is just describing this product's feature, don't you?
i wanna remove other product's name :)

hkdnet committed:
Update README.md

add phrase `each other` and it'll make us and our app cooler :D
and remove a sentence

hkdnet commented:
thanks your advise, @keicy !
please review :+1:

keicy commented:
thanks for your good job! :+1: 
i do merge it.

hkdnet commented:
sorry, this PR did close but was not merged :crying_cat_face:  
i can't figure out why, so i reopen and merge...
please revert if you don't like this @keicy :pray:
